Lib/argparse.py

```diff
@@ -405,8 +405,15 @@ def _format_actions_usage(self, actions, groups):
             else:
                 end = start + len(group._group_actions)
```

Comment:
Please declare at this line
`group_action_count = len(group._group_actions)`

Reply:
Good. I will apply it.